benchmarks.json

```diff
@@ -2,23 +2,29 @@
+        "autotx/tests/agents/token/send/test_send.py::test_auto_tx_send_erc20_parallel": "100.00",
+        "autotx/tests/agents/token/send/test_send.py::test_auto_tx_send_eth_multiple": "100.00",
+        "autotx/tests/agents/token/test_swap.py::test_auto_tx_swap_complex_1": "100.00",
+        "autotx/tests/agents/token/test_swap.py::test_auto_tx_swap_complex_2": "90.00",
+        "autotx/tests/agents/token/test_swap.py::test_auto_tx_swap_multiple_1": "90.00",
+        "autotx/tests/agents/token/test_swap.py::test_auto_tx_swap_multiple_2": "100.00",
+        "autotx/tests/agents/token/test_swap.py::test_auto_tx_swap_native": "100.00",
+        "autotx/tests/agents/token/test_swap.py::test_auto_tx_swap_triple": "100.00",
+        "autotx/tests/agents/token/test_swap.py::test_auto_tx_swap_with_non_default_token": "100.00",
+        "autotx/tests/agents/token/test_swap_and_send.py::test_auto_tx_swap_and_send_complex": "100.00",
+        "autotx/tests/agents/token/test_swap_and_send.py::test_auto_tx_swap_and_send_simple": "100.00",
+        "autotx/tests/agents/token/test_swap_and_send.py::test_auto_tx_send_and_swap_simple": "80.00",
+        "autotx/tests/agents/token/test_swap_and_send.py::test_auto_tx_send_and_swap_complex": "100.00",
         "autotx/tests/agents/token/research/test_research.py::test_get_top_5_memecoins_in_optimism": "90.00",
         "autotx/tests/agents/token/research/test_research.py::test_get_top_5_memecoins": "90.00",
         "autotx/tests/agents/token/research/test_research_and_swap.py::test_auto_tx_research_and_swap_meme_token": "50.00",
-        "autotx/tests/agents/token/test_swap_and_send.py::test_auto_tx_swap_and_send_simple": "100.00",
         "autotx/tests/agents/token/research/test_research.py::test_check_liquidity": "85.00",
-        "autotx/tests/agents/token/send/test_send.py::test_auto_tx_send_eth_multiple": "100.00",
         "autotx/tests/agents/token/research/test_research_and_swap.py::test_auto_tx_research_swap_and_send_governance_token": "70.00",
         "autotx/tests/agents/token/research/test_research.py::test_price_change_information": "90.00",
         "autotx/tests/agents/token/research/test_research.py::test_get_top_5_tokens_from_base": "90.00",
-        "autotx/tests/agents/token/test_swap.py::test_auto_tx_swap_multiple": "100.00",
-        "autotx/tests/agents/token/test_swap_and_send.py::test_auto_tx_swap_and_send_complex": "100.00",
-        "autotx/tests/agents/token/test_swap.py::test_auto_tx_swap_with_non_default_token": "100.00",
         "autotx/tests/agents/token/research/test_research.py::test_get_top_5_most_traded_tokens_from_l1": "70.00",
-        "autotx/tests/agents/token/research/test_research.py::test_get_token_exchanges": "90.00",
-        "autotx/tests/agents/token/send/test_send.py::test_auto_tx_send_erc20_parallel": "100.00"
+        "autotx/tests/agents/token/research/test_research.py::test_get_token_exchanges": "90.00"
     },
     "iterations": 1
```

Comment:
nit: should we add a benchmarks json with 10 iterations?

Reply:
It's not used for anything right now, so I wouldn't bother. Not sure why I implemented it

Reply:
Updated it anyway